### ISSUE TITLE
[BUG] event registration should check if event_route was loaded

### ISSUE THREAD

**Issue by lemenkov** [OpenSIPS/opensips#2822]
Turned out that `even_route[EVENT_NAME] {}` does not check if event_route module is loaded. All user got is a message that
```
Apr 26 15:51:36 INFO:core:evi_publish_event: Registered event <E_UL_CONTACT_INSERT(7)>
``` 
At this point one could assert that everything is ok, and yet the code block will not be triggered since event_route wasn't loaded explicitly in the config file. Either this (event_route loading) has to be done automatically or relevant log message has to be emitted.

**Comment by bogdan-iancu** [comment]
From the logical perspective, I agree with you @lemenkov . Still, from the coding perspective this is a tricky to do, as the core side is just a holder for the `event_route` definitions, nothing more. Only the event_route module is using those routes. And if the module is not loaded, there is no other piece of code in the core to be `event_route` aware, not even able to say if those routes are actually used or not by any other module (as the core is able to check on what the modules are doing).
The solution may be add a new core function, so the "event_route" module would use it when loaded, so the core will know that the routes are used. And if the function is not called, the core will raise a warning as the routes are not used. My 2 cents here. @razvancrainea , what do you think of this?

**Comment by bogdan-iancu** [comment]
PS I do not see this as a bug ;)

**Comment by liviuchircu** [comment]
This was fixed by Vlad in bb9bd920f and backported down to `3.1`.  OpenSIPS will not start anymore if at least one `event_route` is defined, but `event_route.so` not loaded.

**Comment by lemenkov** [comment]
Thank you!
